tests/project.go

```diff
@@ -43,3 +43,107 @@ func TestCreateProject(t *testing.T) {
 	}
 	assert.True(t, sawNewProject)
 }
+
+func TestUpdateProjectDescription(t *testing.T) {
```

Comment:
can you add a truncate projects table statement here? https://github.com/lyft/flyteadmin/blob/master/tests/bootstrap.go#L43 otherwise your tests are not idempotent

Reply:
Yup! Good callout. Had no idea that would be needed.

Reply:
np! it's not really intuitive unless you've worked with the code before